lib/diagnostic.rb

```diff
@@ -78,10 +91,11 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat, and I go 'meow'"
 
 ## Explain why this would be the output, based on the method lookup chain.
-# your response as a comment here
+# The first interpolation would translate over to HouseCat and
```

Comment:
This doesn't quite answer the question since there is no mention of the method lookup chain.  `say_hello` is not defined on `HouseCat`, so Ruby will check the superclass, `Cat`, for it.  Since it's not defined on `Cat` either, it looks up `Cat`'s superclass, `Animal`, finds it, and calls it.